go/vt/vtctl/vdiff2.go

```diff
@@ -54,16 +54,20 @@ func commandVDiff2(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.Fl
 
 	debugQuery := subFlags.Bool("debug_query", false, "Adds a mysql query to the report that can be used for further debugging")
 	onlyPks := subFlags.Bool("only_pks", false, "When reporting missing rows, only show primary keys in the report.")
-	format := subFlags.String("format", "", "Format of report") // "json" or ""
+	var format string
+	subFlags.StringVar(&format, "format", "text", "Format of report") // "json" or "text"
```

Comment:
Any reason why we define `format` for this flag where as everywhere else (including the new `verbose` we use the reference?

Reply:
Avoids having to use two variable names because I call ToLower on this immediately after Parse. I can change it though, not a big deal.

Reply:
👍🏽